src/components/btc-stx-block-links.tsx

```diff
@@ -0,0 +1,102 @@
+import { FC, Fragment } from 'react';
+import { FaBitcoin } from 'react-icons/fa';
+import { BsArrowRightShort } from 'react-icons/bs';
+import { Box, Text, color } from '@stacks/ui';
+import { css } from '@emotion/react';
+import { IconArrowRight } from '@tabler/icons';
+import { buildUrl } from '@components/links';
+import { useRouter } from 'next/router';
+import { StxInline } from '@components/icons/stx-inline';
+import { Circle } from '@components/circle';
+import { useAppSelector } from '@common/state/hooks';
+import { selectActiveNetwork } from '@common/state/network-slice';
+
+interface BtcStxBlockLinksProps {
+  btcBlockHeight?: number;
+  stxBlockHeight: number;
+  stxBlockHash: string;
+  fontSize?: string;
+}
+
+const wrapperStyle = css`
```

Comment:
I am a bit worried that we need to write so much custom CSS with hardcoded values. The component library should be able to hand something as this simple. 

Eventually, we'll need to reconsider our approach to the UI layer on the Explorer.

No action needed, just wanted to share my thought

Reply:
let's discuss this in today's meeting